### DISCUSSION TITLE
Installation error: Status 503 Egress is over the account limit.

### DISCUSSION THREAD

**Discussion by sparkminh** [community/community#44633] [npm]
### Select Topic Area

Product Feedback

### Body

When I use npm install sharp, it shows error below.
```
npm ERR! code 1
npm ERR! path /Volumes/work/me/Boost/boost-websites-171-arnz/node_modules/sharp
npm ERR! command failed
npm ERR! command sh -c (node install/libvips && node install/dll-copy && prebuild-install) || (node install/can-compile && node-gyp rebuild && node install/dll-copy)
npm ERR! sharp: Downloading https://github.com/lovell/sharp-libvips/releases/download/v8.13.3/libvips-8.13.3-darwin-arm64v8.tar.br
npm ERR! sharp: Please see https://sharp.pixelplumbing.com/install for required dependencies
npm ERR! sharp: Installation error: Status 503 Egress is over the account limit.
```

This is no npm issue and command shows error on downloading package from github.
I have seen this discussion here, https://github.com/lovell/sharp-libvips/issues/121 .On that time it was resolved by github support team but the error happens again now.
Can you please help me with this?

Thank you.

**Comment by cehfisher** [comment]
Thanks for posting in the GitHub Community, @sparkminh! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've asked @queenofcorgis to move this issue into the correct category for you. Good luck!
